scripts/install.sh

```diff
@@ -0,0 +1,22 @@
+#check if verbose flag is used
+if [[ $* == -v ]]
+then
+set -o xtrace
```

Comment:
I would suggest setting other flags, especially `-e`. [This gist](https://gist.github.com/mohanpedala/1e2ff5661761d3abd0385e8223e16425) offers an easy explanation.